backend/Shared/QueueService/QueueServices/DefaultConnectionFactory.cs

```diff
@@ -2,12 +2,12 @@
 {
     public class DefaultConnectionFactory : RabbitMQ.Client.ConnectionFactory
     {
-        public DefaultConnectionFactory()
+        public DefaultConnectionFactory(string hostName = "localhost")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/app/iamprovidence/CVRPWT/pullRequest?prid=4635982)